js/templates/modals/settings/moderation.html

```diff
@@ -29,23 +32,137 @@ <h2 class="h3 clrT">Moderation</h2>
           </div>
           <div class="col9">
-            <% if (ob.errors.description) print(ob.formErrorTmpl({ errors: ob.errors.description })) %>
-            <textarea rows="3" maxlength="160" name="modInfo.description" id="settingsModeratorDescription"
-                      class="clrBr clrSh2" placeholder="Describe yourself"><%= ob.description %></textarea>
+            <% if (ob.errors['modInfo.fee.feeType']) print(ob.formErrorTmpl({ errors: ob.errors['modInfo.fee.feeType'] })) %>
+            <% if (ob.errors['modInfo.fee.feeTypeNoFee']) print(ob.formErrorTmpl({ errors: ob.errors['modInfo.fee.feeTypeNoFee'] })) %>
+            <% if (ob.errors['modInfo.fee.feeTypeNoPercent']) print(ob.formErrorTmpl({ errors: ob.errors['modInfo.fee.feeTypeNoPercent'] })) %>
+            <% if (ob.errors['modInfo.fee.fixedFee.feeTypeNoCurrency']) print(ob.formErrorTmpl({ errors: ob.errors['modInfo.fee.fixedFee.feeTypeNoCurrency'] })) %>
+            <% if (ob.errors['modInfo.fee.fixedFee.feeTypeNoAmount']) print(ob.formErrorTmpl({ errors: ob.errors['modInfo.fee.fixedFee.feeTypeNoAmount'] })) %>
+            <div class="flexRow gutterH rowTn">
+              <div class="col4">
+                <select id="moderationFeeType" name="modInfo.fee.feeType" class="clrBr clrP clrSh2">
+                  <option value="PERCENTAGE" <% if (!ob.fee || (ob.fee && ob.fee.feeType === 'PERCENTAGE')) print('selected') %>>
+                    <%= ob.polyT('settings.moderationTab.percentage') %>
+                  </option>
+                  <option value="FIXED" <% if (ob.fee && ob.fee.feeType === 'FIXED') print('selected') %>>
+                    <%= ob.polyT('settings.moderationTab.fixed') %>
+                  </option>
+                  <option value="FIXED_PLUS_PERCENTAGE" <% if (ob.fee && ob.fee.feeType === 'FIXED_PLUS_PERCENTAGE') print('selected') %>>
+                    <%= ob.polyT('settings.moderationTab.fixedPlusPercentage') %>
+                  </option>
+                </select>
+              </div>
+                <div class="col2 js-feePercentageInput <% if (ob.fee && ob.fee.feeType === 'FIXED') { print('visuallyHidden') } %>">
+                  <div class="inputPercentWrapper clrBr clrSh2">
+                    <% var perc = ob.fee.percentage || 0; %>
+                    <input type="number"
+                           name="modInfo.fee.percentage"
+                           value="<%= perc %>"
+                           data-var-type="number">
+                  </div>
+                </div>
+                <div class="col2 js-feeFixedInput <% if (!ob.fee || (ob.fee && ob.fee.feeType === 'PERCENTAGE')) { print('visuallyHidden') } %>">
+                  <% var amt = ob.fee && ob.fee.fixedFee ? ob.fee.fixedFee.amount : 0;
+                     var ccode = ob.fee && ob.fee.fixedFee ? ob.fee.fixedFee.currencyCode : ob.defaultCurrency;
+                     amt =  amt ? ob.formatPrice(amt, ccode === 'BTC') : 0;
```

Comment:
If someone enters the price as a string, formatPrice will bomb and prevent the error message from the fixedFee model from showing:

![image](https://cloud.githubusercontent.com/assets/794517/22380209/6a8326bc-e471-11e6-9a0e-30a1dbb291da.png)

I think the fix is to only pipe the amount through formatPrice if it's a number, otherwise just let it render as a string and the model validation will catch it:

![image](https://cloud.githubusercontent.com/assets/794517/22380249/92a87714-e471-11e6-8152-197d2b380e66.png)